include/cutlass/gemm/collective/sm100_mma_array_warpspecialized_emulated.hpp

```diff
@@ -1092,7 +1092,8 @@ struct CollectiveMma<
     cute::tma_descriptor_fence_acquire(get<1>(input_tensormaps));
   }
 
-private:
+protected:
```

Comment:
Does this file only `emulates` F32-precision with `9xBF16` or does emulate any other computation too?

Reply:
just bf16x9